configuration/debian/tedge_agent/postrm

```diff
@@ -26,14 +9,7 @@ purge_agent_directory() {
 
 case "$1" in
     purge)
-        remove_tedge_agent_user
-        remove_sudoers_file
-        purge_agent_directory
-    ;;
-
-    remove)
-        remove_tedge_agent_user
-        remove_sudoers_file
+       purge_agent_directory
     ;;
 
     upgrade|failed-upgrade|abort-install|abort-upgrade|disappear)
```

Comment:
Maybe this is smarter. Regarding my comment above.
```suggestion
    remove|upgrade|failed-upgrade|abort-install|abort-upgrade|disappear)
```

Reply:
added